docker-compose/hotrod/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.7'
```

Comment:
We already have this in examples/hotrod/docker-compose.yml

Reply:
my image tag is different that is why I create a different file 
`localhost:5000/jaegertracing/example-hotrod:${GITHUB_SHA:-latest}`

Reply:
this is not the reason to duplicate the docker compose file. You should instead parameterize the main one.